examples/install_with_clone_centos7.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+error() {
+  printf '\E[31m'; echo "$@"; printf '\E[0m'
+}
+
+red=$'\e[1;31m'
+grn=$'\e[1;32m'
+blu=$'\e[1;34m'
+end=$'\e[0m'
+myip=$(hostname -I)
+
+if [[ ! $EUID -eq 0 ]]; then
+    error "This script should be run using sudo or as the root user"
+    exit 1
+fi
+
+curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+
+# Install remi repo
+yum install -y http://rpms.remirepo.net/enterprise/remi-release-7.rpm
+
+# Install mariadb 10.4 repo
+cat >/etc/yum.repos.d/MariaDB.repo<<EOF 
+[mariadb]
+name = MariaDB
+baseurl = http://yum.mariadb.org/10.4/centos7-amd64
+gpgkey=https://yum.mariadb.org/RPM-GPG-KEY-MariaDB
+gpgcheck=1
+EOF
+
+# Install Dependecies
+yum --enablerepo=remi-safe,remi-php72 install -y httpd php php-cli php-json php-mysqlnd php-dom php-curl unzip zip nodejs git mod_ssl MariaDB-server MariaDB-compat
+
+# Enable apache and mariadb, start mariadb
+systemctl enable httpd
+systemctl enable mariadb
+systemctl start mariadb
+
+# install pm2
+npm -g install pm2
+
+#Download composer
+curl -sS https://getcomposer.org/installer -o composer-setup.php
+# Install
+php composer-setup.php --install-dir=/usr/local/bin --filename=composer
+
+# Move to www & clone repository
+cd /var/www || exit
+
+git clone https://github.com/trenb/LxdMosaic.git
+
+mkdir -p /var/www/LxdMosaic/src/sensitiveData/certs
+chown -R apache:apache /var/www/LxdMosaic/src/sensitiveData/certs
+
+# Move in LxdManager
+cd /var/www/LxdMosaic || exit
+
+npm install
+
+# Install Dependecies
+composer install
+
+cp .env.dist .env
+
+# Update env values
+## DB Host
+sed -i -e 's/DB_HOST=/DB_HOST=localhost/g' .env
+## DB User
+sed -i -e 's/DB_USER=/DB_USER=lxd/g' .env
+## DB Pass
+sed -i -e 's/DB_PASS=/DB_PASS=lxdManagerPasswordComplex321/g' .env
+
+# Import data into mysql
+mysql < sql/users.sql
+mysql < sql/seed.sql
+mysql < sql/0.1.0.sql
+mysql < sql/0.2.0.sql
+mysql < sql/fleet_storage_usage.sql
+
+
+# Install a self-signed certificate as CentOS doesn't ship with one
+mkdir -p /etc/ssl/private
+openssl req -new -newkey rsa:2048 -days 3650 -nodes -x509 \
+            -subj "/C=NA/ST=NoState/L=NoLocale/O=Blah/CN=mosaic" \
+            -keyout /etc/ssl/private/ssl-cert-snakeoil.key  -out /etc/ssl/certs/ssl-cert-snakeoil.pem
+
+cp examples/lxd_manager_centos.conf /etc/httpd/conf.d/
+
+pm2 start node/events.js
+
+pm2 startup
+
+pm2 save
+
+# Add cron job for gathering data
+crontab -l | { cat; echo "*/5 * * * * php /var/www/LxdMosaic/src/cronJobs/fleetAnalytics.php"; } | crontab -
+crontab -l | { cat; echo "*/1 * * * * php /var/www/LxdMosaic/src/cronJobs/hostsOnline.php"; } | crontab -
+
+systemctl restart httpd
+
+printf "${grn}\nInstallation successfull \n\n"
+printf  "You now need to point your browser at ${blu}https://${myip}:3000${end} ${red}and accept the self signed certificate${end} \n\n"
```

Comment:
This is a clever a trick to show the ip addresses but its not entirely clear  that the user needs to go to `https://myip:3000` if they have an ipv4 & ipv6 address

```suggestion
printf  "You now need to point your browser at ${blu}https://myip:3000${end} ${red}and accept the self signed certificate${end} \n\n"
```

Reply:
Why is that even necessary? I haven't had to do that step on any of my installs. I navigate directly to the webui. 

Anyhow, you can emit the end notification however, I'm not married to the current wording!

Reply:
Interestingly I think you may be right I seem to have fixed that issue with the apache reverse proxy (thinking about it i may have done it when I did the console update) if we could leave it as it comes in the ubuntu script ill be happy

Reply:
Unless you want to merge the two scripts together, I see no reason to make changes to your ubuntu script.

Reply:
Sorry i mean can you remove your clever ip list trick so it reads "https://myip:3000" like it does in the ubuntu script

Reply:
I'm happy dropping the ```myip``` variable stuff, since I have no way of knowing for certain if the user is on ipv4 or ipv6, and it's probably not correct to assume ipv4 only.

Reply:
I've removed it.

Reply:
Cool if you can;

-  commit the change
- Update the changelog
- Change the target repo

Ill merge this and the patch in a second :)

Reply:
Should be good.